coldfront/plugins/qumulo/tests/test_tasks.py

```diff
@@ -245,3 +260,284 @@ def test_conditionally_update_storage_allocation_statuses_checks_all_pending_all
             self.assertEqual(
                 conditionally_update_storage_allocation_status_mock.call_count, 2
             )
+
+
+class TestIngestQuotasWithDailyUsages(TestCase):
+    def setUp(self) -> None:
+        self.original_storage_path = os.environ.get("STORAGE2_PATH")
+        self.STORAGE2_PATH = "/storage2/fs1"
+        os.environ["STORAGE2_PATH"] = self.STORAGE2_PATH
+
+        self.client = Client()
```

Comment:
It's not needed here, good call.

It actually only needs used in views and maybe forms, I think?

> A class that can act as a client for testing purposes.
> 
>     It allows the user to compose GET and POST requests, and
>     obtain the response that the server gave to those requests.
>     The server Response objects are annotated with the details
>     of the contexts and templates that were rendered during the
>     process of serving the request.
> 
>     Client objects are stateful - they will retain cookie (and
>     thus session) details for the lifetime of the Client instance.
> 
>     This is not intended as a replacement for Twill/Selenium or
>     the like - it is here to allow testing against the
>     contexts and templates produced 

by a view, rather than the
    HTML rendered to the end-user.